explainer.md

```diff
@@ -0,0 +1,59 @@
+# Server Timing Explained
+
+## What's All This About?
+Server Timing allows developers to monitor the performance of server-handling of requests of every resource that makes up a page by overloading Navigation (for the base HTML) and Resource Timing (for scripts, images, etc). This new API enables developers to have insight into performance bottlenecks of the `Request` block below, that is, after `requestStart` and before `responseStart`.
+ 
+![navigation timing image](./nav-timing.png)
+
+Adhering to a standard response header format gives developer tools the ability to visualize backend timers that the developer (or intermediate proxies) deem important. The Javascript API gives analytics vendors the ability to collect and beacon more detailed timing data about each request. 
```

Comment:
"to a response header format defined by this specification gives developers" -- more explicit that we're defining it, not someone else.